security/yubi/wsl2/setup-wsl2-ssh-pageant.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# TODO: fork the repository or integrate contents here to ensure availability
+windows_destination="/mnt/c/Users/Public/Downloads/wsl2-ssh-pageant.exe"
```

Comment:
shouldn't we add these files to the c:\wsl... so that we can rn this script for newly installed guest systems without wsl2 is located in a downloads folder?